Sources/Plasma/PubUtilLib/plAvatar/plPhysicalControllerCore.cpp

```diff
@@ -344,14 +344,14 @@ void plAnimatedMovementStrategy::IRecalcAngularVelocity(float elapsed, hsMatrix4
 
     float angleSincePrev = AngleRad2d(curForward.fX, curForward.fY, prevForward.fX, prevForward.fY);
     bool sincePrevSign = angleSincePrev > 0.0f;
-    if (angleSincePrev > float(M_PI))
-        angleSincePrev = angleSincePrev - TWO_PI;
+    if (angleSincePrev > hsConstants::pi<float>)
+        angleSincePrev = angleSincePrev - hsConstants::two_pi<float>;
```

Comment:
Looks like this one got missed somehow?
```suggestion
        angleSincePrev -= hsConstants::two_pi<float>;
```